prototype/southbound/mqtt/mqtt-device-factory/src/test/resources/configs/config.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
Have you considered using https://github.com/osgi/osgi-test/tree/main/org.osgi.test.junit5.cm instead of the configurator? It keeps the configuration closed to the test code that's using it.

Reply:
Thanks for the example, I've updated the PRs